### PR TITLE
Disabled data addition.

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asab/library/providers/zookeeper.py [hunk 1]
```diff
@@ -31,23 +32,23 @@ def __init__(self, app, path):
...
 	async def finalize(self, app):
 		# close client
 		await self.Zookeeper._stop()
 
-	async def _on_zk_ready(self, event_name, zkcontainer):
-		if zkcontainer == self.ZookeeperContainer:
-			self.Zookeeper = self.ZookeeperContainer.ZooKeeper
-
 	async def read(self, path):
 		if self.Zookeeper is None:
```
Comment (mithunbharadwaj) [inline asab/library/providers/zookeeper.py:48]: @ateska 

Can I know why we throw this warning ?

This will be caught as an exception correct?

Comment (ateska) [reply asab/library/providers/zookeeper.py:57]: The exception text is very misleading.

**Review by ateska — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Review by ateska — CHANGES_REQUESTED** [review]
Concept of `tenant` should not be used in asab.
It is abstraction from higher levels.

File: asab/library/service.py [hunk 2]
```diff
@@ -36,9 +45,9 @@ async def read(self, path):
 			if item is not None:
 				return item
 
-	async def list(self, path, recursive=True):
+	async def list(self, path, tenant, recursive=True):
```
Comment (ateska) [inline asab/library/service.py:48]: `tenant` should not be introduced here - or at least optionally, if you need that for "disable" evaluation.

**Review by ateska — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

File: asab/library/service.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ async def read(self, path):
 			if item is not None:
 				return item
 
-	async def list(self, path, tenant=None, recursive=True):
+	async def list(self, path, tenant=None, recursive=False):
```
Comment (ateska) [inline asab/library/service.py:59]: Provide a documentation for parameters, please.

**Review by ateska — CHANGES_REQUESTED** [review]
Please add a documentation string, then mergeable.

**Review by ateska — APPROVED** [review]

**Pull request merged**